app/src/main/res/values-be/strings.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="app_name">Сөзлик</string>
+
+    <string name="navigation_drawer_open">Open navigation drawer</string>
+    <string name="navigation_drawer_close">Close navigation drawer</string>
+
+    <string name="android_studio">Android Studio</string>
+    <string name="android_studio_android_com">android.studio@android.com</string>
+    <string name="dictionary">Сөзлик</string>
+    <string name="translation">Аўдарма</string>
+    <string name="favourites">Сайландылар</string>
+    <string name="history">Тарийх</string>
+    <string name="settings">Сазлаўлар</string>
+    <string name="about">Ҳаққымызда</string>
+    <string name="search_hint">Излеген сөзиңизди киритиң</string>
+    <string name="suggestion_found">\"%1$s\" табылмады, бәлким төмендегилерди излеген шығарсыз.</string>
+    <string name="suggestion_not_found">\"%1$s\" бойынша нәтийжелер табылмады.</string>
+    <string name="share_translation">Сөзди бөлисиў</string>
+    <string name="share">Бөлисиў</string>
+    <string name="about_text">Бул бағдарлама <b>Шағала Лаб</b> топары ҳәм оның ағзалары <a href="mailto:manasbazarbayev@gmail.com">Манас Базарбаев</a> ҳәм <a href="mailto:kalabaevshj@gmail.com">Шарапат Қалабаев</a> тәрепинен ислеп шығылды. Бағдарлама бойынша пикир яки усыныс билдириу ушын <a href="mailto:info@shagalalab.com">info@shagalalab.com</a> электрон почтасына хат жазың ямаса <a href="http://www.shagalalab.com" rel="nofollow">www.shagalalab.com</a> сайтында "байланыс" бетинде жайласқан форманы толтырың.\n\n<b>Shaǵala Lab</b> - Шағала Лаб Қарақалпақ тилиндеги бағдарламалар лабораториясы\n\nБизлердиң жумысларымыз бенен <a href="https://play.google.com/store/apps/developer?id=Shag%27ala+Lab" rel="nofollow">Google Play</a> яки <a href="http://www.shagalalab.com/" rel="nofollow">сайтымызда</a> танысыу имкәниятына ийесиз.</string>
+    <string-array name="pref_alphabet_options">
+        <item>Latın</item>
+        <item>Кирилл</item>
+    </string-array>
+    <string-array name="pref_alphabet_values">
+        <item>@string/pref_alphabet_latin</item>
+        <item>@string/pref_alphabet_cyrillic</item>
+    </string-array>
+    <string name="pref_alphabet_latin">latin</string>
+    <string name="pref_alphabet_cyrillic">cyrillic</string>
```

Comment:
mina ekewin original xml'da tomendegige ozgeriwin kerek, bul jerde bolmawi kerek bul:
    `<string name="pref_alphabet_latin">qq</string>`
    `<string name="pref_alphabet_cyrillic">be</string>`

Reply:
usi zatti na'zerde tutip edim keshe men cyrillic qa qoyganimda app qayaqtan biledi be ni paydalaniwimiz kerek ekenin dep, men be ni cyrillic qilip o'zgertiriw kerekpeken dep oylap edim